assets/src/dashboard/app/views/exploreTemplates/content/index.js

```diff
@@ -58,7 +58,12 @@ function Content({
     <Layout.Scrollable>
       <FontProvider>
         <TransformProvider>
-          <UnitsProvider pageSize={view.pageSize}>
+          <UnitsProvider
+            pageSize={{
+              width: view.pageSize.width,
+              height: view.pageSize.height,
+            }}
```

Comment:
Did this object have other props or was this just to be more specific?

Reply:
I added in `containerHeight` and if we pass it in to the pageSize on unitsProvider we get console warnings. Rather than update the propType on the edit-story side of things I opted to do this where necessary so that the props are kept tight.